### PR TITLE
Ensure we only set a maintainer once

### PR DESCRIPTION
**Pull request by thommay**

### CODE REVIEW

**Comment by nellshamrell** [comment]
@thommay are you still working on this one?

**Comment by thommay** [comment]
Can this wait for a while? I need to do some other things and then come back and keep going on this.

**Comment by thommay** [comment]
@nellshamrell can we resurrect this one?

File: db/schema.rb [hunk 1]
```diff
@@ -11,11 +11,10 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150512134228) do
+ActiveRecord::Schema.define(version: 20150626120818) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-  enable_extension "pg_trgm"
```
Comment (nellshamrell) [inline db/schema.rb]: Any reason for removing this extension?

Comment (thommay) [inline db/schema.rb]: i assume rails did that when i ran `rake db:migrate`, because it certainly wasn't intentional.

Comment (smith) [inline db/schema.rb]: Only if you want to make search break :)

Comment (nellshamrell) [inline db/schema.rb]: Can you re-add it?  Otherwise this looks good.

**Comment by nellshamrell** [comment]
:+1: Merging :)

**Pull request merged**